File: LES_EXAMENS/EXAM_PYTHON_NORMAL_2024_2025_MIP/exerciceF.c
#include <stdio.h>
int M[3][3];
int main(){
    for(int i=0;i<3;i++){
        for(int j=0;j<3;j++){
            printf("saisir la case {%d}{%d}: ",i+1,j+1);
            scanf("%d",&M[i][j]);
        }
    }
    for(int i=0;i<3;i++){
        for(int j=0;j<3;j++){
            printf(" %d ",M[j][i]);
        }
        printf("\n");
    }
    return 0;
}

File: LES_EXAMENS/EXAM_PYTHON_NORMAL_2024_2025_MIP/exerciceF.py
M=[[None for i in range(3)] for i in range(3)]
for i in range(0,3):
    for j in range(0,3):
        M[i][j]=int(input(f"saisir la case {i+1} {j+1}: "))
for i in range(0,3):
    for j in range(0,3):
        print(f" {M[j][i]} ",end="")
    print("")
